pom.xml

```diff
@@ -103,6 +103,16 @@
       <artifactId>commons-httpclient</artifactId>
       <version>3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-collections4</artifactId>
+        <version>4.1</version>
+    </dependency>
```

Comment:
commons-collections version 3 doesn't support generics. commons-collections version 3 goes indirectly from other dependency. 

I'd like to avoid generics warning, so I had to new a new, 4th version. It means, we need a new dependency.